requests_futures/sessions.py

```diff
@@ -109,3 +109,73 @@ def close(self):
         if self._owned_executor:
             self.executor.shutdown()
 
+    def get(self, url, **kwargs):
+        r"""
+        Sends a GET request. Returns :class:`Future` object.
+
+        :param url: URL for the new :class:`Request` object.
+        :param \*\*kwargs: Optional arguments that ``request`` takes.
+        :rtype : concurrent.futures.Future
```

Comment:
Stinks there doesn't seem to be a way to indicate what the Future objects `.result()` will return here.